### DISCUSSION TITLE
How does .env file is created during CI build?

### DISCUSSION THREAD

**Discussion by cris-m** [nickjj/docker-flask-example#9] [Q&A] [answered]
I was trying to run CI build but I keep on getting the following error:
```
sed: can't read .env*: No such file or directory
```

**Comment by nickjj** [comment, marked as answer]
https://github.com/nickjj/docker-flask-example/blob/a9a02f9fb4092e8a704fcbcf5fe1da0ab801bb77/run#L150 copies the example file to `.env`.

**Reply by cris-m** [reply]
Thanks I found that in `run` file. The problem was my folder structure. 
Now I can not access `bin` folder and the following line throw one error:
```
shellcheck run bin/*
```
Here is an error. 
```
run: run: openBinaryFile: does not exist (No such file or directory)
```

**Reply by nickjj** [reply]
If you're using this example app it exists. The `bin/` directory is there.

Maybe you need to install the dependencies for CI: https://github.com/nickjj/docker-flask-example/blob/a9a02f9fb4092e8a704fcbcf5fe1da0ab801bb77/run#L133

This is all defined in: https://github.com/nickjj/docker-flask-example/blob/main/.github/workflows/ci.yml

**Reply by cris-m** [reply]
I totally get it. It was my problem. Since `run` file was not on the root of the repo, I had to add `  cd "$(dirname "$0")"` to set the folder where `run` script is executing as working directory.
